### PR TITLE
docs: update resource path to relative supporting v6 doc path

### PR DESCRIPTION
**Pull request by wattachai-lseg**
## Description

update resource paths to relative ones supporting /v6 path on prod envrionment 

## Type of change
- [x] a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: documents/src/pages/start/browser-support.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ You should include polyfills and transpile the code into `ES5` to support old br
 
 <table>
 <tbody><tr>
-<td align="center"><img src="/resources/images/chrome.png"></img></td>
-<td align="center"><img src="/resources/images/safari.png"></img></td>
-<td align="center"><img src="/resources/images/firefox.png"></img></td>
-<td align="center"><img src="/resources/images/edge.png"></img></td>
-<td align="center"><img src="/resources/images/electron.png"></img></td>
-<td align="center"><img src="/resources/images/ie.png"></img></td>
+<td align="center"><img src="resources/images/chrome.png"></img></td>
```
Comment (Nantawat-Poothong) [inline documents/src/pages/start/browser-support.md:15]: Why do these image paths not start with `./resources` like another

File: documents/src/templates/import-element/ef-autosuggest.md [hunk 2]
```diff
@@ -6,6 +6,6 @@ name: autosuggest
  * Import paths and theme-loader used for demonstration purposes
  */
```
Comment (Nantawat-Poothong) [inline documents/src/templates/import-element/ef-autosuggest.md:7]: Interesting, can you go to the autosuggest section Header and footer?
I found a white background demo, don't know which purpose to set the white background on it

**Review by Nantawat-Poothong — CHANGES_REQUESTED** [review]
LGTM & I also leave a few comments

File: documents/src/pages/start/browser-support.md (hunk 1 above)
Comment (wattachai-lseg) [reply documents/src/pages/start/browser-support.md:15]: path in img's `src` would be relative if there is no leading `/`. [MDN examples](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#examples) use this pattern as well.

File: documents/src/templates/import-element/ef-autosuggest.md (hunk 2 above)
Comment (wattachai-lseg) [reply documents/src/templates/import-element/ef-autosuggest.md:7]: background color removed

**Review by Nantawat-Poothong — APPROVED** [review]

**Pull request merged**